src/modules/users/CheckPWQuality.cpp

```diff
@@ -31,20 +31,23 @@
 #include <memory>
 
 PasswordCheck::PasswordCheck()
-    : m_message()
+    : m_message( [](){ return ""; } )
```

Comment:
That lambda should probably be a free function (so it's easier to add APIDOX to it explainng that it's a MessageFunc that doesn't complain), and return QString() rather than "".